packages/cli/src/commands/network/parameters.ts

```diff
@@ -0,0 +1,106 @@
+import { BaseCommand } from '../../base'
+
+export default class Parameters extends BaseCommand {
+  static description = 'View network parameters'
+
+  static flags = {
+    ...BaseCommand.flags,
+  }
+
+  async run() {
+    const goldToken = await this.kit.contracts.getGoldToken()
+    const stableToken = await this.kit.contracts.getStableToken()
+    const attestations = await this.kit.contracts.getAttestations()
+    console.log(
+      'attestations.attestationExpirySeconds',
+      (await attestations.attestationExpirySeconds()).toString(10),
+      'seconds'
+    )
+    console.log(
+      'attestations.attestationExpiryFee (gold)',
+      (await attestations.attestationRequestFees(goldToken.address)).toString(10),
+      ''
+    )
+    console.log(
+      'attestations.attestationExpiryFee (stable)',
+      (await attestations.attestationRequestFees(stableToken.address)).toString(10),
+      ''
+    )
+
+    const lockedGold = await this.kit.contracts.getLockedGold()
+    console.log(
+      'lockedGold.maxNoticePeriod',
+      (await lockedGold.maxNoticePeriod()).toString(10),
+      'seconds'
+    )
+
+    const sortedOracles = await this.kit.contracts.getSortedOracles()
+    console.log(
+      'sortedOracles.reportExpirySeconds',
+      (await sortedOracles.reportExpirySeconds()).toString()
+    )
+
+    const exchange = await this.kit.contracts.getExchange()
```

Comment:
what about adding a method to the exchange wrapper called: `getConfiguration(): Promise<ExchangeConfig>` that returns all this?

The same applies to all other Contract wrappers.

Reply:
In fact, we can do even better.

```ts
kit.getCeloConfiguration(): Promise<CeloConfiguration>

interface CeloConfig: {
    exchange: ExchangeConfig,
    .... all other contracts...
}
```